tests/utilities/test_data.py

```diff
@@ -194,16 +194,16 @@ def __init__(self, attribute2, *args, **kwargs):
     assert dataloader.batch_size == 2
     assert dataloader.at1 == "attribute1"  # But the value still gets passed when it should
 
-    dataloader = _update_dataloader(dataloader, dataloader.sampler)
+    updated_dataloader = _update_dataloader(dataloader, dataloader.sampler)
```

Comment:
nit: is the name change necessary? you should try to minimize changes in long PRs like this one

Reply:
It isn't, but it is change only on the commit level, not on the PR level